source/privacy.html.erb

```diff
@@ -46,221 +46,175 @@ title: Privacy notice
-        <p class="govuk-body">You have the right to:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>information about how your personal data is processed</li>
-          <li>a copy of that personal data</li>
-          <li>that anything inaccurate in your personal data is corrected immediately</li>
-        </ul>
-
-        <p class="govuk-body">You can also:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>raise an objection about how your personal data is processed</li>
-          <li>request that your personal data is erased if there is no longer a justification for it</li>
-          <li>ask that the processing of your personal data is restricted in certain circumstances</li>
-        </ul>
-
-        <p class="govuk-body">If you have any of these requests, get in contact with our Data Protection Officer - the
-          contact details are at the bottom of the page.</p>
-
-        <h2 class="govuk-heading-m" id="changes-to-this-notice">Changes to this notice</h2>
-        <p class="govuk-body">We may change this privacy notice. In that case the ‘last updated’ date at the top of this
-          page will also change. Any changes to this privacy notice will apply to you and your data immediately. If
-          these changes affect how your personal data is processed, GDS will take reasonable steps to make sure you
-          know.</p>
-
-        <h2 class="govuk-heading-m" id="questions-and-complaints">Questions and complaints</h2>
-        <p class="govuk-body">Contact
-          <a class="govuk-link" href="mailto:gds-privacy-office@digital.cabinet-office.gov.uk">gds-privacy-office@digital.cabinet-office.gov.uk</a>
-          if you either:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>have any questions about anything in this document</li>
-          <li>think that your personal data has been misused or mishandled</li>
-        </ul>
-
-        <p class="govuk-body">
-          You can also contact the Cabinet Office Data Protection Officer.<br />
-          <a class="govuk-link" href="mailto:DPO@cabinetoffice.gov.uk">DPO@cabinetoffice.gov.uk</a><br />
-          Cabinet Office<br />
-          70 Whitehall<br />
-          London SW1A 2AS
-        </p>
-
-        <p class="govuk-body">If you have a complaint, you can also contact the
-          <a class="govuk-link" href="https://ico.org.uk/">Information Commissioner</a>, who is an independent regulator
-          set up to uphold information rights.</p>
-
-        <p class="govuk-body">
-          <a class="govuk-link" href="mailto:casework@ico.org.uk">casework@ico.org.uk</a><br />
-          0303 123 1113<br />
-          Wycliffe House<br />
-          Water Lane<br />
-          Wilmslow<br />
-          Cheshire SK9 5AF
-        </p>
-      </div>
+      <h1 class="govuk-heading-l" id="heading-1">Privacy notice</h1>
+      <p class="govuk-body-l">Last updated: 17 January 2023</p>
```

Comment:
We need to be careful about changing the content on this page - it's been reviewed by IA as well as content folks. We should only change the date if the content has changed (which I don't think we should do, below - including the heading level).

Reply:
this has actually been updated recently, I didn't change the date. See `master`